packages/ag-charts-community/src/chart/themes/chartTheme.ts

```diff
@@ -599,7 +616,8 @@ export class ChartTheme {
         properties.set(DEFAULT_FONT_FAMILY, 'Verdana, sans-serif');
         properties.set(DEFAULT_LABEL_COLOUR, 'rgb(70, 70, 70)');
         properties.set(DEFAULT_MUTED_LABEL_COLOUR, 'rgb(140, 140, 140)');
-        properties.set(DEFAULT_AXIS_GRID_COLOUR, 'rgb(219, 219, 219)');
+        properties.set(DEFAULT_AXIS_GRID_COLOUR, 'rgb(224,234,241)');
+        properties.set(DEFAULT_PIE_SERIES_LABEL_COLOUR, 'white');
```

Comment:
Can you make sure to add a `DarkTheme` update for this too?

Reply:
Will adjust this when I move on to working on darkTheme?